ppocr/data/imaug/fce_aug.py

```diff
@@ -208,7 +208,7 @@ def generate_crop_target(self, image, all_polys, pad_h, pad_w):
         for polygon in all_polys:
             rect = cv2.minAreaRect(polygon.astype(np.int32).reshape(-1, 2))
             box = cv2.boxPoints(rect)
-            box = np.int0(box)
+            box = np.int64(box)
```

Comment:
please explain the reason for making this change. it seems irrelevant to the purpose of the PR.